lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -1920,10 +1577,7 @@ function lessThan($value)
  */
 function lessThanOrEqual($value)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$value". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)